src/porepy/models/constitutive_laws.py

```diff
@@ -4189,33 +3626,23 @@ def friction_coefficient(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         )
 
 
-class ShearDilation:
+class ShearDilation(PorePyModel):
     """Class for calculating fracture dilation due to tangential shearing.
 
     The main method of the class is :meth:`shear_dilation_gap`.
 
     """
 
-    nd: int
-    """Ambient dimension of the problem. Normally set by a mixin instance of
-    :class:`porepy.models.geometry.ModelGeometry`.
-
-    """
-    tangential_component: Callable[[list[pp.Grid]], pp.ad.Operator]
-    """Operator giving the tangential component of vectors. Normally defined in a mixin
-    instance of :class:`~porepy.models.models.ModelGeometry`.
-
-    """
-    plastic_displacement_jump: Callable[[list[pp.Grid]], pp.ad.Operator]
+    displacement_jump: Callable[[list[pp.Grid]], pp.ad.Operator]
```

Comment:
I don't see the displacement_jump in this class?

Reply:
Removed